### PR TITLE
chore: add rest server

### PR DESCRIPTION
**Pull request by amsmith-pro**
Signed-off-by: AM Smith <smitham42@gmail.com>

### CODE REVIEW

File: client-grpc/src/lib.rs [hunk 1]
```diff
@@ -1,4 +1,20 @@
...
+//! <div align="center">
+//!     <a href="https://github.com/Arrow-air/lib-ccsds/releases">
+//!         <img src="https://img.shields.io/github/v/release/Arrow-air/lib-ccsds?include_prereleases" alt="GitHub release (latest by date including pre-releases)">
+//!     </a>
+//!     <a href="https://github.com/Arrow-air/lib-ccsds/tree/main">
+//!         <img src="https://github.com/arrow-air/lib-ccsds/actions/workflows/rust_ci.yml/badge.svg?branch=main" alt="Rust Checks">
+//!     </a>
+//!     <a href="https://discord.com/invite/arrow">
+//!         <img src="https://img.shields.io/discord/853833144037277726?style=plastic" alt="Arrow DAO Discord">
+//!     </a>
```
Comment (GoodluckH) [inline client-grpc/src/lib.rs]: some links seem to be pointing to lib-ccsds

File: server/src/main.rs [hunk 2]
```diff
@@ -1,14 +1,64 @@
...
+//!         <img src="https://img.shields.io/github/v/release/Arrow-air/lib-ccsds?include_prereleases" alt="GitHub release (latest by date including pre-releases)">
+//!     </a>
+//!     <a href="https://github.com/Arrow-air/lib-ccsds/tree/main">
+//!         <img src="https://github.com/arrow-air/lib-ccsds/actions/workflows/rust_ci.yml/badge.svg?branch=main" alt="Rust Checks">
+//!     </a>
+//!     <a href="https://discord.com/invite/arrow">
+//!         <img src="https://img.shields.io/discord/853833144037277726?style=plastic" alt="Arrow DAO Discord">
+//!     </a>
+//!     <br><br>
+//! </div>
```
Comment (GoodluckH) [inline server/src/main.rs]: same here

File: server/src/main.rs [hunk 3]
```diff
@@ -26,31 +76,131 @@ impl SvcTelemetryRpc for SvcTelemetryImpl {
...
+/// Starts the REST API server for this microservice
+pub async fn rest_server(grpc_clients: GrpcClients) {
+    let rest_port = std::env::var("DOCKER_PORT_REST")
+        .unwrap_or_else(|_| "8000".to_string())
+        .parse::<u16>()
+        .unwrap_or(8000);
+
+    let app = Router::new()
+        .fallback(not_found.into_service())
+        .route("/tlm/adsb", routing::post(rest_api::adsb))
```
Comment (GoodluckH) [inline server/src/main.rs]: should we spell out telemetry for readability?

**Review by GoodluckH — DISMISSED** [review]

File: client-grpc/src/lib.rs (hunk 1 above)
Comment (amsmith-pro) [reply client-grpc/src/lib.rs]: nice catch!

https://github.com/Arrow-air/svc-telemetry/pull/5/commits/26dbe0229d09340013fc1986e94dade4d3a69c15

File: server/src/main.rs (hunk 2 above)
Comment (amsmith-pro) [reply server/src/main.rs]: https://github.com/Arrow-air/svc-telemetry/pull/5/commits/26dbe0229d09340013fc1986e94dade4d3a69c15

File: server/src/main.rs (hunk 3 above)
Comment (amsmith-pro) [reply server/src/main.rs]: https://github.com/Arrow-air/svc-telemetry/pull/5/commits/26dbe0229d09340013fc1986e94dade4d3a69c15

**Review by GoodluckH — APPROVED** [review]

**Pull request merged**